pkgs/top-level/php-packages.nix

```diff
@@ -420,7 +420,7 @@ lib.makeScope pkgs.newScope (self: with self; {
         }
         {
           name = "intl";
-          buildInputs = [ icu64 ];
+          buildInputs = [ icu73 ];
```

Comment:
What about switching to just `icu`. So it would be always up to date. wdym @drupol

Reply:
Would this cause any breakage?

Reply:
I don't know anything about this thing sorry...

Reply:
let's keep it so then :)